scripts/install-helpers/baremetal-coco/intel-dcap/registration-ds.yaml.in

```
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pccs-registration
+  namespace: intel-dcap
+type: Opaque
+data:
+  USER_TOKEN: ${USER_TOKEN}
+  PCCS_URL: ${PCCS_URL}
+  SECURE_CERT: ${SECURE_CERT}
+  HTTP_PROXY: ${REGISTRATION_HTTP_PROXY}
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-dcap-registration-flow
+  namespace: intel-dcap
+spec:
+  selector:
+    matchLabels:
+      name: intel-dcap-registration-flow
+  template:
+    metadata:
+      labels:
+        name: intel-dcap-registration-flow
+    spec:
+      nodeSelector:
+        intel.feature.node.kubernetes.io/sgx: 'true'
+      automountServiceAccountToken: false
+      containers:
+      - name: intel-dcap-registration-flow
+        securityContext:
+          readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: true
+          privileged: true
```

we could explain the reason for privileged here too:
```suggestion
          privileged: true # Required to access host efivarfs
```

a future improvement could be to move the privileged part to an `initContainer`